Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.11
+
+WORKDIR /openduck-py
+
+COPY openduck-py/requirements.txt /openduck-py/
+
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y uvicorn awscli && \ 
+    apt-get clean
+
+RUN pip install --upgrade pip
+RUN pip install Cython
```

Comment:
Can this be added to requirements.txt?

Reply:
its a linux package not a python package

Reply:
for some reason it only works if its specifically installed here